tests/conformance/bindings/bindings.go

```diff
@@ -36,44 +39,67 @@ func (tc *TestConfig) createInvokeRequest() bindings.InvokeRequest {
 	}
 }
 
-func ConformanceTests(t *testing.T, props map[string]string, binding bindings.OutputBinding, config TestConfig) {
+func ConformanceTests(t *testing.T, props map[string]string, inputBinding bindings.InputBinding, outputBinding bindings.OutputBinding, config TestConfig) {
+	// We need to increase the number of processes as the input binding is its own process that kicks off multiple sub processes
+	runtime.GOMAXPROCS(20)
```

Comment:
@yaron2 Can you give your thoughts on this? Why is it that GOMAXPROCS needs an increase?

Reply:
I'm not sure we need a change: `GOMAXPROCS` by default sets it's value to the number of CPU cores on a machine. It used to be `1` in older versions of Go.

Reply:
Is there an alternate to adjusting it? The behavior I was seeing was (with debug statements):

Test starts
goroutine for consumption is kicked off
Test waits
Test completes
goroutine executes

It had the same behavior every time and increasing the max procs allowed the routine to run in parallel with the test.

Reply:
were you seeing this behavior on your local machine?

Reply:
Yes, I haven't tried with the provisioned workflow.

Reply:
the test and the goroutine are running on the same process?

Reply:
This is where I may not be well versed enough in how go does things under the hood. My gut instinct would say that a goroutine would just be a managed thread vs. go creating a new process. 

That being said, I can at least say that the test is what is kicking off these goroutines.

Reply:
Go will not be creating a new process, but a goroutine is "sort of" a managed thread. Goroutines are scheduled on top of OS system threads, which are set with GOMAXPROCS and default to the number of CPUs your box has. I find it very weird that the goroutine executed only after the test completes.

Reply:
I agree, which is what lead me down the 'how many things does go run' path and we ended up here. Is this something that we really shouldn't be doing? Or is it just weird? If there's a better way I'd love to learn it cause I'm also not a huge fan of setting runtime like that in a test.

Reply:
Circling back to this, I just got AzureStorageQueues to work. Unlike the Kafka binding, they only spin up a single goroutine. This did work without the max procs setting (though I left it in). It does seem to back that it was a goroutine limit though which is why I left it in in case other bindings had issues. Thoughts @yaron2? Given the currently implemented binding works without it I'm fine removing it, but I worry we'll just pass the buck down.